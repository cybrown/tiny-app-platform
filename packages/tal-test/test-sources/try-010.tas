fun toto1() {
    missing_variable
}

fun toto2() {
    toto1() catch {
        "value from toto2"
    }
}

toto2()
