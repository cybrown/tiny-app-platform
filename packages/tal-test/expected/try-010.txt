(2:5) Unknown symbol: missing_variable
any
